finder-app/manual-linux.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
 # Script outline to install and build kernel.
-# Author: Siddhant Jajoo.
+# Author: Parth Varsani
+# Date: February 2nd, 2025
 
 set -e
 set -u
 
 OUTDIR=/tmp/aeld
 KERNEL_REPO=git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git
-KERNEL_VERSION=v5.1.10
+KERNEL_VERSION=v5.15.163
 BUSYBOX_VERSION=1_33_1
-FINDER_APP_DIR=$(realpath $(dirname $0))
+FINDER_APP_DIR=/home/temp/aesd/assignment-1-Parthvarsani13/finder-app
```

Comment:
@Parthvarsani13 this won't work with the automated test runner, you need to have a relative path based on scriptname.   Looks like this was fixed later with https://github.com/cu-ecen-aeld/assignments-3-and-later-Parthvarsani13/commit/f3423f72529e442965a8a93837df12719b34813f